recipes/containers-sugar/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "containers-sugar" %}
+{% set version = "1.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/containers_sugar-{{ version }}.tar.gz
+  sha256: 28c9ed69d61a52016065af927f00eea147eb148ff383c313543da1e9514a15c6
+
+build:
+  skip: True  # [win]
```

Comment:
Same comment as https://github.com/conda-forge/staged-recipes/pull/24382/files#r1384945123

Reply:
@ocefpaf thank you for the review!

question: containers-sugar works on top of docker-compose/docker .. does it work on windows?
I checked the docker-compose feedstock and I am very surprised that it doesn't have any skip for windows :O
do you know if I would need to have wsl to test that on windows?

Reply:
Maybe it is better to use the OS-noarch then. With the __unix virtual package.

Reply:
got it .. thank you so much! appreciate that!